api/swagger.yaml

```diff
@@ -0,0 +1,152 @@
+swagger: "2.0"
+info:
+  description: "Just a dictionary for **\"Галицька\"** language."
+  version: "1.0.0"
+  title: "Dictionary"
+externalDocs:
+  description: "GitHub"
+  url: "https://github.com/yushni/dictionary"
+host: "localhost"
+basePath: "/"
+tags:
+- name: "word"
+  description: ""
+schemes:
+- "http"
+paths:
+  /word:
+    post:
+      tags:
+      - "word"
+      summary: "Add a new word to the dictionary"
+      description: ""
+      operationId: "addWord"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Metadata of the word that have to be added to the dictionary"
+        required: true
+        schema:
+          $ref: "#/definitions/Word"
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            type: "object"
+            properties:
+              id:
+                type: "integer"
+                format: "int64"
+        "405":
```

Comment:
405 status code means method not allowed (e.g. used when you try to send **GET** request but you only **POST** allowed) 
https://developer.mozilla.org/ru/docs/Web/HTTP/Status/405 

I think we can use 400 Bad Request